src/Input/Command.php

```diff
@@ -52,6 +52,8 @@ class Command extends Parser implements Groupable
 
     protected ?string $_alias = null;
 
+    protected string $logo = '';
```

Comment:
should be _logo as the command will have magic getter for user input args that may collide

Reply:
Done!